spec/index.bs

```diff
@@ -417,7 +419,7 @@ dictionary GPUColorStateDescriptor {
 
     required GPUBlendDescriptor alphaBlend;
     required GPUBlendDescriptor colorBlend;
-    GPUColorWriteFlags writeMask = GPUColorWriteBits.ALL;
+    GPUColorWriteFlags writeMask = 15;  // GPUColorWriteBits.ALL
```

Comment:
Ew :(

Reply:
Please at least use 0x0f or 0xf?

Reply:
Done here and in the bits definitions too.

Reply:
And GPUTextureUsage and all the other bits definitions.